test/quantization/core/test_quantized_tensor.py

```diff
@@ -328,6 +329,82 @@ def test_qtensor_dtypes(self):
             rqr = qr.dequantize()
             self.assertTrue(np.allclose(r.numpy(), rqr.numpy(), atol=2 / scale))
 
+    @unittest.skipIf(not TEST_CUDA, "No gpu is not available.")
+    def test_qtensor_to_device(self):
+        dtype_and_zero_types = [
+            (torch.quint8, torch.float),
+            (torch.qint8, torch.float),
+            #  (torch.qint32, torch.float) not supported for quantize_per_channel
+            (torch.quint8, torch.long),
+            (torch.qint8, torch.long),
+            (torch.qint32, torch.long),
+        ]
+        axis = 1
+        device = torch.device('cuda')
+        for dtype, zero_type in dtype_and_zero_types:
+            r = torch.rand(2, 2, dtype=torch.float) * 10
+            scales = torch.rand(2).abs()
+            zero_points = (torch.rand(2) * 10).round().to(zero_type)
+
+            # dqr = torch.quantize_per_channel(r, scales, zero_points, axis, dtype)
+            # dqr = dqr.to(device)
+            # dqr_cuda = torch.quantize_per_channel(r.to(device), scales.to(
+            #     device), zero_points.to(device), axis, dtype)
+            # dqr_cuda = dqr_cuda.to('cpu')
+            # self.assertEqual('cuda', dqr.device.type)
+            # self.assertEqual('cpu', dqr_cuda.device.type)
```

Comment:
why comment these out?

Reply:
I split the pr into multiple parts, so the tests would fail at this point since this pr doesn't have the per channel functionality implemented

Reply:
Second jerry's comment, will be good to have a per-channel test and per-tensor test for .to device functionality.

Reply:
ok, I can do this.